lib/_http_outgoing.js

```diff
@@ -575,10 +586,13 @@ OutgoingMessage.prototype.setHeader = function setHeader(name, value) {
   validateHeaderName(name);
   validateHeaderValue(name, value);
 
+  if (name.toLowerCase() === 'content-length') {
+    this._contentLength = value;
+  }
+
   let headers = this[kOutHeaders];
   if (headers === null)
     this[kOutHeaders] = headers = ObjectCreate(null);
-
```

Comment:
Unrelated change

Reply:
+1. By the way, are changes in `doc/api/errors.md` also required? It seems like most of them are unrelated to an issue as well.